plugins/source/aws/resources/services/timestream/databases_mock_test.go

```diff
@@ -32,7 +33,10 @@ func buildTimestreamDatabasesMock(t *testing.T, ctrl *gomock.Controller) client.
 	m.EXPECT().ListTagsForResource(gomock.Any(), gomock.Any(), gomock.Any()).Return(
 		&timestreamwrite.ListTagsForResourceOutput{Tags: tags}, nil)
 
-	return client.Services{Timestreamwrite: m}
+	return client.Services{
+		AWSConfig:       aws.Config{Region: "us-east-1"},
+		Timestreamwrite: m,
+	}
```

Comment:
That works or you could do this:

```
	services.Regions = []string{testOpts.Region}
	services.AWSConfig.Region = testOpts.Region
	c.accountMutex["testAccount"] = &sync.Mutex{}
```

In `AwsMockTestHelper`

Reply:
921261c4694dd98c0ff4a6316de89549736484a8